birdr/view.py

```diff
@@ -172,3 +172,10 @@ def show(checklist_name: str) -> None:
             branch.add(f"{mark} {species}")
 
     rich.print(tree)
+
+
+@main.command()
+@click.argument("version", required=False)
```

Comment:
Why do I have to have an argument in @click.argument() in order for the app to run?

Reply:
`@click.argument()` is only required if the function needs to take an argument from the command line. You can remove the function argument (since it's unused) and get rid of `@click.argument()`, leaving just `@main.command()`.